app/models/patient.rb

```diff
@@ -1,10 +1,11 @@
 class Patient < ActiveRecord::Base
   has_many :episode_of_cares
 
-  validates_format_of :first_name, :with => /\A[^0-9`!@#\$%\^&*+_=]+\z/, message: "No numbers or special chars"
-  validates_format_of :last_name, :with => /\A[^0-9`!@#\$%\^&*+_=]+\z/, message: "No numbers or special chars"
-  validates :scido_id, numericality: { only_integer: true, greater_than: 0}
-  validates_format_of :ssn, :with => /\d{3}-\d{2}-\d{4}/, message: "Expect format 111-22-3333"
+  validates_format_of :first_name, :with => /\A[^0-9`!@#\$%\^&*+_=]+\z/, message: "Enter the patient's first name"
```

Comment:
This will print the message "Enter the patient's first name" if you type in something in like "f1r$tname".  That seems a little less than ideal.

Filing bug.